test/unit/core/diffusion_test.cc

```diff
@@ -1104,7 +1104,7 @@ TEST(DiffusionTest, EulerNeumannNonZeroBoundaries) {
   // ----------------------------------------------------------
 
   double intermediate_concentration_backup = 0.0;
-  int tot = 10000;
+  int tot = 1000;
```

Comment:
I think this modification is good.

Reply:
Looks good to me as well, total diffusion time should be enough for the test.